graders/r/r_autograder/run.sh

```diff
@@ -56,7 +56,7 @@ if [ ${DEBUG} == "on" ]; then ls -ld ${MERGE_DIR} ${MERGE_DIR}/*; fi
 # RUN
 ##########################
 
-cd ${MERGE_DIR}
+cd ${MERGE_DIR} || exit
```

Comment:
https://www.shellcheck.net/wiki/SC2164

Reply:
I've long wondered if we should just be using `set -e` on these scripts. From the docs:

> Having a `set -e` command anywhere in the script will disable this message, even though it won't necessarily prevent the issue.

Why wouldn't that be sufficient to prevent the issue?

Reply:
That feels like a better fix, we should use that instead.

Reply:
I'll leave doing so for future PRs.